### DISCUSSION TITLE
Howto create a fork including pending pull requests?

### DISCUSSION THREAD

**Discussion by palasinio** [community/community#21841] [New to GitHub]
<p>Hi,</p>
<p>I’m new to this all.</p>
<p>Without having a developper environment I just tried to fork this repository:</p>
<p><a href="https://github.com/SphtKr/homebridge-zway" rel="nofollow noopener">https://github.com/SphtKr/homebridge-zway</a></p>
<p>There are 5 pulls pending.</p>
<p>How can I get a fork including this pulls via github webpage?</p>
<p>regards,</p>
<p>Chris.</p>

**Comment by seveas** [comment, marked as answer]
<p>I’m afraid that that’s not possible in one go, you will need to recreate the pull requests.</p>
